commands/shell.go

```diff
@@ -0,0 +1,89 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/c-bata/go-prompt"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionoscloudsdk/comptplus"
+	"github.com/spf13/viper"
+)
+
+var advancedPrompt = &comptplus.CobraPrompt{
+	RootCmd:                  rootCmd.Command,
+	ShowHelpCommandAndFlags:  true,
+	DisableCompletionCommand: true,
+	AddDefaultExitCommand:    true,
+	ShowHiddenCommands:       true,
+	ShowHiddenFlags:          true,
+	GoPromptOptions: []prompt.Option{
+		prompt.OptionTitle("ionosctl"),
+		prompt.OptionPrefix("> "),
+		prompt.OptionShowCompletionAtStart(),
+
+		prompt.OptionAddKeyBind(prompt.KeyBind{Key: prompt.ShiftLeft, Fn: prompt.GoLeftWord}),
+		prompt.OptionAddKeyBind(prompt.KeyBind{Key: prompt.ShiftRight, Fn: prompt.GoRightWord}),
+		prompt.OptionAddKeyBind(prompt.KeyBind{Key: prompt.ShiftDown, Fn: prompt.GoLineBeginning}),
```

Comment:
I think the shortcuts are a bit counter-intuitive. You would expect `CTRL+Left/Right` to allow you to move from one word to another, and for `Home` and `End` to get you to the beginning/end of the line (these are functional within the shell at the moment, but still).

Reply:
sadly `Home` and `End` don't work within the shell instance, I agree with `CTRL + Left / Right` though.

Reply:
Why don't `Home` and `End` work? I replaced them locally and it worked alright.

Reply:
on zsh `?[23~` and `?[24~` are entered instead of their respective funcs being executed. However we can just leave the default go-prompt controls I guess, and return to this custom controls topic at a later date when we can allow the user to customize his controls.